### PR TITLE
Make the table presentation more informative by default

### PR DESCRIPTION
**Pull request by mavit**
By default, mention the server and channel names in the presentation message.

Addresses #19.

### CODE REVIEW

File: src/bot_logic.py [hunk 1]
```diff
@@ -170,6 +170,14 @@ async def create_bga_game(message, bga_account, game, players, p1_id, options):
         return
     valid_bga_players = []
     invited_players = []
+
+    default_presentation = 'Created by BGA Discord Bot'
```
Comment (pocc) [inline src/bot_logic.py:174]: If memory serves, presentation is limited to 240 bytes (irrespective of number of unicode characters).

Why not keep the presentation default line above and use an fstring like so?

```
# global variable
# shortened from https://discord.com/api/oauth2/authorize?client_id=711844812424216598&permissions=79872&scope=bot
INVITE_URL = "https://bit.ly/3nHY12z"
# line 
f"Generated for {message.author.name} on Discord server {message.guild.name} with the BGA Discord Bot. Invite to your server with {INVITE_URL}."
```

Are you sending make game requests directly to the bot?

Comment (mavit) [reply src/bot_logic.py:174]: Iʼve just tested it, and using the form on the website, I was able to upload 2kB without any problem (although it discarded most of this when I clicked on the `[modify]` button).  This seems good enough for our purposes.

Sending requests of any kind directly to the bot seems like something that users will reasonably try to do, and `!bga make …` does currently work in a direct message.  However, in this case, `message.guild` is unset.

Comment (pocc) [reply src/bot_logic.py:174]: I think I meant to say 60 bytes - yeah that's the limit after you hit modify.

Comment (pocc) [reply src/bot_logic.py:174]: If the user messages the bot, I think that should force interactive mode to get missing pieces like what channel to post the game to (the embed is most useful when other people can see it).

Comment (mavit) [reply src/bot_logic.py:174]: Sounds like something to revisit when interactive mode is fleshed out.

For interactive mode, Iʼm imagining building a list of channels that both the user and the bot are members of, to offer as potential choices.  I feel like “no channel” would be a valid option to include on this list.

Comment (pocc) [reply src/bot_logic.py:174]: Setup channel is partially implemented. Take a look at the interactive !setup.

**Comment by pocc** [comment]
I've added this as the default presentation: `Created by the discord BGA bot (https://github.com/pocc/bga_discord)`. You can also change your personal presentation if you want.

Are we good to close this for the time being?

**Comment by mavit** [comment]
Looks good!

**Pull request closed without merging**